packages/material-ui/src/styles/createMuiTheme.js

```diff
@@ -36,6 +36,7 @@ function createMuiTheme(options = {}, ...args) {
       spacing,
       shape,
       transitions,
+      variants: {},
```

Comment:
Did you mean to give it a default value i.e. 
```suggestion
      variants = {},
```
?

Looks like it isn't needed anyway (all tests pass) so 
```suggestion
      variants,
```
should be preferred

Reply:
Actually the initial was correct, as this is part of object definition, and I am just initializing here the variants to be an empty object. Tests are failing without it

Reply:
You are not initializing `variants` to be an empty object. In fact `let { variants: {}, ...rest } = props` would throw if `props.variants` is `undefined`.

Reply:
nevermind we're not destructuring here :laughing: